pkg/domain/infosync/info.go

```diff
@@ -209,7 +209,10 @@ func GlobalInfoSyncerInit(
 	if err != nil {
 		return nil, err
 	}
-	is.labelRuleManager = initLabelRuleManager(etcdCli)
+	if pdHTTPCli != nil {
+		pdHTTPCli = pdHTTPCli.WithRespHandler(pdResponseHandler)
```

Comment:
Could we put it to `initLabelRuleManager`? It seems we use `pdHTTPCli` only in `initLabelRuleManager`.

Reply:
Later we will integrate the PD HTTP client into the Placement Rule manager below also, so I keep it initialized here.